client/js/views/templates/Footer.js

```diff
@@ -1,14 +1,88 @@
-module.exports = function() { return `` +
-`<footer>
-    <div>
-        <span>Contact</span>
-        <span>
-            <a href="mailto:info@futuredays.io">info@futuredays.io</a>
-        </span>
-    </div>
-    <div>
-        <span>Copyright</span>
-        <span>${new Date().getFullYear()} FutureDays Software</span>
-    </div>
-</footer>`
+/**
+* Project: Future Days Farm Website
+* Created By: Chris Baron
+* Date Last Modified: 1/19/2018 by Alex Cadigan
+* Description: This file builds the footer elements on the Farm Website homepage
+*/
+
+module.exports = function() 
+{ 
+    return `<footer id = "footerBody">
```

Comment:
I should have gave you more information before you began building.

Don't use `id` attributes on elements.  Use `data-js` instead.  This will allow you to automatically reference the element in the view file.

In this case, I don't even think you need to use `data-js` unless you are going to utilize the element in the view.

Reply:
A class of `Footer` is automatically applied to the top level element of an view.